app/src/main/java/com/example/ithaca_transit_android_v2/NetworkingAdapters.kt

```diff
@@ -1,11 +1,13 @@
 package com.example.ithaca_transit_android_v2
 
-import com.example.ithaca_transit_android_v2.models.Coordinate
-import com.example.ithaca_transit_android_v2.models.Location
-import com.example.ithaca_transit_android_v2.models.LocationType
+import com.example.ithaca_transit_android_v2.models.*
 import com.squareup.moshi.FromJson
+import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 import com.squareup.moshi.ToJson
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
let's avoid making this generic import call, and just import the class that you need.

Reply:
please remove `java.util.*`